challenge_2/python/OkraArmband/README.md

```diff
@@ -0,0 +1,5 @@
+The program "unique_ele.py" has two methods in it: unique() and unique_v2(). Both of these functions carry out the same purpose and will function identically.
```

Comment:
Please try to add how to run this file, as well as test inputs and outputs for someone can fully understand what this program is doing. Good use of everything else though!